mapproxy/source/ogcapimaps.py

```diff
@@ -0,0 +1,189 @@
+                raise ex
+
+            for mimetype in ("image/png", "image/jpeg"):
+                href = find_href_in_links(
+                    j["links"],
+                    "http://www.opengis.net/def/rel/ogc/1.0/map",
+                    mimetype,
+                )
+                if href:
+                    self.map_format_to_href[
+                        mimetype[len("image/"):]
+                    ] = self._build_url(href)
+
+            supported_srs = set()
+            if "crs" in j:
+                for crs_url in j["crs"]:
+                    crs_code = normalize_srs_code(ogc_crs_url_to_auth_code(crs_url))
+                    self.map_crs_code_to_url[crs_code] = crs_url
+                    supported_srs.add(crs_code)
+            if "storageCrs" in j:
+                crs_url = j["storageCrs"]
+                crs_code = normalize_srs_code(ogc_crs_url_to_auth_code(crs_url))
+                self.map_crs_code_to_url[crs_code] = crs_url
+                supported_srs.add(crs_code)
+
+            if len(supported_srs) == 0:
+                self.map_crs_code_to_url[
+                    "EPSG:4326"
+                ] = "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+                supported_srs.add("EPSG:4326")
+
+            self.supported_srs = SupportedSRS([SRS(crs) for crs in supported_srs])
+
+    def get_map(self, query):
+        self._get_maps_list()
+        return WMSLikeSource.get_map(self, query)
+
+    def _retrieve(self, query, format):
+        url = self.map_format_to_href[format]
+        req = BaseRequest(url=url)
+        if query.srs.srs_code != "EPSG:4326" and query.srs.is_axis_order_ne:
+            req.params["bbox"] = "%.17g,%.17g,%.17g,%.17g" % (
+                query.bbox[1],
+                query.bbox[0],
+                query.bbox[3],
+                query.bbox[2],
+            )
+        else:
+            req.params["bbox"] = "%.17g,%.17g,%.17g,%.17g" % (
+                query.bbox[0],
+                query.bbox[1],
+                query.bbox[2],
+                query.bbox[3],
+            )
+        if query.srs.srs_code != "EPSG:4326":
+            req.params["bbox-crs"] = self.map_crs_code_to_url[query.srs.srs_code]
+        if not (
+            query.srs.srs_code == "EPSG:4326"
+            and len(self.supported_srs.supported_srs) == 1
+        ):
+            req.params["crs"] = self.map_crs_code_to_url[query.srs.srs_code]
+        req.params["width"] = query.size[0]
+        req.params["height"] = query.size[1]
+        if self.image_opts.transparent:
+            req.params["transparent"] = "true"
+        if self.transparent_color:
+            if len(self.transparent_color) == 4:
+                r, g, b, a = self.transparent_color
+                req.params["bgcolor"] = "0x%02X%02X%02X%02X" % (a, r, g, b)
+            else:
+                r, g, b = self.transparent_color
+                req.params["bgcolor"] = "0x%02X%02X%02X" % (r, g, b)
```

Comment:
transparent_color is used in mapproxy to specify a color that should be output by mapproxy as transparent (by internal image manipulation). See `WMSLikeSource`, mehtod `make_transparent`.

The bgcolor on the opposite is used to specify a color for transparent / no data values in the source, that should be painted in the given color.

The code can be removed, as the handling of transparent_color should be handled already in the parent class.
Or this could be rewritten to just support and passthrough the parameter `bgcolor` from the user config to the source as is.